### PR TITLE
Fix spelling typos in documentation

### PR DESCRIPTION
**Pull request by anshulahuja98**
A minor PR to fix spelling typos in the comments documentation for the code.

### CODE REVIEW

**Comment by bieniusa** [comment]
Thanks a lot!

File: src/clocksi_interactive_coord.erl [hunk 1]
```diff
@@ -245,7 +245,7 @@ finish_op(From, Key, Result) ->
 %%    txid: transaction id handled by this fsm, as defined in src/antidote.hrl.
 %%    updated_partitions: the partitions where update operations take place.
 %%    num_to_ack: when sending prepare_commit,
-%%                number of partitions that have acked.
+%%                number of partitions that have asked.
```
Comment (ergl) [inline src/clocksi_interactive_coord.erl]: This is actually not a typo. In any case, it could be changed to "acknowledged"

**Comment by anshulahuja98** [comment]
@ergl @bieniusa 
Should I make a fresh PR rectifying the same
Sorry for the mistake

**Comment by bieniusa** [comment]
No, I took care of it. Thanks!

**Pull request merged**